packages/ra-core/src/routing/useRestoreScrollPosition.ts

```diff
@@ -1,7 +1,8 @@
 import { useEffect } from 'react';
-import { useStore } from '../store';
-import { debounce } from 'lodash';
```

Comment:
This change is unrelated to the PR, but I noticed that the bundles for the demos were embarking the entire lodash, so I figured I'd fix this regression.